_stbt/core.py

```diff
@@ -1948,28 +1946,25 @@ def pull_frame(self, timeout_secs=10):
 
     def frames(self, timeout_secs=None):
         import time
-        self.start_timestamp = None
+        if timeout_secs is not None:
+            end_time = time.time() + timeout_secs
 
         with self.lock:
             while True:
                 ddebug("user thread: Getting sample at %s" % time.time())
-                sample = self._get_sample(max(10, timeout_secs))
+                sample = self._get_sample(max(10, timeout_secs or 0))
```

Comment:
This is an unrelated change. It's also unnecessary:

```
>>> max(10, None)
10
```

But it doesn't hurt.

Reply:
I'll undo it in a fixup

Reply:
Done